src/components/layout/MainNavbar/NavbarTranslate.js

```diff
@@ -1,35 +1,34 @@
 import React, { useState } from "react";
 import i18next from "i18next";
-import {
-  Button,
-} from "shards-react";
+import { Button } from "shards-react";
 import "flag-icon-css/css/flag-icon.min.css";
 import { Store } from "../../../flux";
 
 const languages = Store.getLanguages();
 const countryCode = Store.getCountryCode();
 
 const NavbarTranslate = () => {
-  const [flag, setFlag] = useState(false)
+  const [flag, setFlag] = useState(false);
 
   const toggleLocale = () => {
-    setFlag(!flag)
-    if(flag)
-    // English
-      i18next.changeLanguage(languages[0])
-    else
+    setFlag(!flag);
+    if (flag)
+      // English
+      i18next.changeLanguage(languages[0]);
     // Malay
-      i18next.changeLanguage(languages[1])
-  }
-  
+    else i18next.changeLanguage(languages[1]);
+  };
 
   return (
     <div className="navbar-translate-btn">
       <Button onClick={toggleLocale}>
         <span
-          className={`flag-icon flag-icon-${!flag ? countryCode[0] : countryCode[1]} p-1`}
-        ></span></Button>
-      </div>
+          className={`flag-icon flag-icon-${
+            flag ? countryCode[1] : countryCode[0]
```

Comment:
Try to keep this in 1 line as before. Usually its easier to read that way for HTML codes.